src/routes/InventoryHostStaleness.js

```diff
@@ -9,11 +9,11 @@ import { usePermissionsWithContext } from '@redhat-cloud-services/frontend-compo
 import { GENERAL_HOST_STALENESS_READ_PERMISSION } from '../components/InventoryHostStaleness/constants';
 import { Page, PageSection } from '@patternfly/react-core';
 import HostStalenessNoAccess from '../components/InventoryHostStaleness/HostStalenessNoAccess';
-import { GENERAL_GROUPS_READ_PERMISSION } from '../constants';
+import { GENERAL_HOSTS_READ_PERMISSIONS } from '../constants';
 
 const REQUIRED_PERMISSIONS = [
   GENERAL_HOST_STALENESS_READ_PERMISSION,
-  GENERAL_GROUPS_READ_PERMISSION,
+  GENERAL_HOSTS_READ_PERMISSIONS,
 ];
 
 const HostStaleness = () => {
```

Comment:
For the permissions check, you should use `true` for checkAll parameter (the second one); this is due to the fact that both permissions should be fulfilled:

```
  const { hasAccess: canReadHostStaleness } = usePermissionsWithContext(
    REQUIRED_PERMISSIONS,
    true,
    false
  );
```

Reply:
Another thing: I also think we have to change the `checkResourceDefinitions` (the third parameter) to true or just remove it (because it set by default to true): the reason is that according to miro graphs, the whole card should be visible only if users have inventory:hosts:read permission to all hosts. With ignoring RD, you are basically saying "it's sufficient if user has any hosts:read permission even when they are limited with RD, e.g. to some group).

Reply:
good catch, than you!